### ISSUE TITLE
Eval bug: Seed-OSS crash after typing user promt

### ISSUE THREAD

**Issue by ndrewpj** [ggml-org/llama.cpp#15547]
### Name and Version

version: 6246 (b1ab9182)
built with cc (Ubuntu 11.4.0-1ubuntu1~22.04) 11.4.0 for x86_64-linux-gnu

### Operating systems

Linux

### GGML backends

Vulkan

### Hardware

AMD Ryzen Ai 395 Max+ Pro 128GB

### Models

unsloth/Seed-OSS-36B-Instruct-GGUF:Q6_K

### Problem description & steps to reproduce

After loading the model I try to chat with it - it crashed after my first user promt

### Relevant log output

```shell
~/Downloads/llama_vulkan/build/bin/llama-cli -ngl 999 -fa --temp 0.6 --top-k 20 --top-p .95 --min-p 0 --repeat-penalty 1.2 --no-mmap -m ~/Downloads/Seed-OSS-36B-Instruct-Q6_K.gguf --no-warmup
load_backend: loaded RPC backend from /home/andrey/Downloads/llama_vulkan/build/bin/libggml-rpc.so
ggml_vulkan: Found 1 Vulkan devices:
ggml_vulkan: 0 = AMD Radeon Graphics (RADV GFX1151) (radv) | uma: 1 | fp16: 1 | bf16: 0 | warp size: 64 | shared memory: 65536 | int dot: 1 | matrix cores: KHR_coopmat
load_backend: loaded Vulkan backend from /home/andrey/Downloads/llama_vulkan/build/bin/libggml-vulkan.so
load_backend: loaded CPU backend from /home/andrey/Downloads/llama_vulkan/build/bin/libggml-cpu-icelake.so
build: 6246 (b1ab9182) with cc (Ubuntu 11.4.0-1ubuntu1~22.04) 11.4.0 for x86_64-linux-gnu
main: llama backend init
main: load the model and apply lora adapter, if any
llama_model_load_from_file_impl: using device Vulkan0 (AMD Radeon Graphics (RADV GFX1151)) - 72341 MiB free
llama_model_loader: loaded meta data with 39 key-value pairs and 771 tensors from /home/andrey/Downloads/Seed-OSS-36B-Instruct-Q6_K.gguf (version GGUF V3 (latest))
llama_model_loader: Dumping metadata keys/values. Note: KV overrides do not apply in this output.
llama_model_loader: - kv   0:                       general.architecture str              = seed_oss
llama_model_loader: - kv   1:                               general.type str              = model
llama_model_loader: - kv   2:                               general.name str              = Seed-Oss-36B-Instruct
llama_model_loader: - kv   3:                           general.finetune str              = Instruct
llama_model_loader: - kv   4:                           general.basename str              = Seed-Oss-36B-Instruct
llama_model_loader: - kv   5:                       general.quantized_by str              = Unsloth
llama_model_loader: - kv   6:                         general.size_label str              = 36B
llama_model_loader: - kv   7:                            general.license str              = apache-2.0
llama_model_loader: - kv   8:                           general.repo_url str              = https://huggingface.co/unsloth
llama_model_loader: - kv   9:                   general.base_model.count u32              = 1
llama_model_loader: - kv  10:                  general.base_model.0.name str              = Seed OSS 36B Instruct
llama_model_loader: - kv  11:          general.base_model.0.organization str              = ByteDance Seed
llama_model_loader: - kv  12:              general.base_model.0.repo_url str              = https://huggingface.co/ByteDance-Seed...
llama_model_loader: - kv  13:                               general.tags arr[str,3]       = ["vllm", "unsloth", "text-generation"]
llama_model_loader: - kv  14:                       seed_oss.block_count u32              = 64
llama_model_loader: - kv  15:                    seed_oss.context_length u32              = 524288
llama_model_loader: - kv  16:                  seed_oss.embedding_length u32              = 5120
llama_model_loader: - kv  17:               seed_oss.feed_forward_length u32              = 27648
llama_model_loader: - kv  18:              seed_oss.attention.head_count u32              = 80
llama_model_loader: - kv  19:           seed_oss.attention.head_count_kv u32              = 8
llama_model_loader: - kv  20:                    seed_oss.rope.freq_base f32              = 10000000,000000
llama_model_loader: - kv  21:  seed_oss.attention.layer_norm_rms_epsilon f32              = 0,000001
llama_model_loader: - kv  22:              seed_oss.attention.key_length u32              = 128
llama_model_loader: - kv  23:            seed_oss.attention.value_length u32              = 128
llama_model_loader: - kv  24:                       tokenizer.ggml.model str              = gpt2
llama_model_loader: - kv  25:                         tokenizer.ggml.pre str              = seed-coder
llama_model_loader: - kv  26:                      tokenizer.ggml.tokens arr[str,155136]  = ["<seed:bos>", "<seed:pad>", "<seed:e...
llama_model_loader: - kv  27:                  tokenizer.ggml.token_type arr[i32,155136]  = [3, 3, 3, 4, 4, 4, 4, 4, 4, 3, 3, 3, ...
llama_model_loader: - kv  28:                      tokenizer.ggml.merges arr[str,154737]  = ["Ġ Ġ", "Ġ t", "i n", "Ġ a", "e r...
llama_model_loader: - kv  29:                tokenizer.ggml.bos_token_id u32              = 0
llama_model_loader: - kv  30:                tokenizer.ggml.eos_token_id u32              = 2
llama_model_loader: - kv  31:            tokenizer.ggml.padding_token_id u32              = 1
llama_model_loader: - kv  32:                    tokenizer.chat_template str              = {# Unsloth Chat template fixes #}\n{# ...
llama_model_loader: - kv  33:               general.quantization_version u32              = 2
llama_model_loader: - kv  34:                          general.file_type u32              = 18
llama_model_loader: - kv  35:                      quantize.imatrix.file str              = Seed-OSS-36B-Instruct-GGUF/imatrix_un...
llama_model_loader: - kv  36:                   quantize.imatrix.dataset str              = unsloth_calibration_Seed-OSS-36B-Inst...
llama_model_loader: - kv  37:             quantize.imatrix.entries_count u32              = 448
llama_model_loader: - kv  38:              quantize.imatrix.chunks_count u32              = 149
llama_model_loader: - type  f32:  321 tensors
llama_model_loader: - type q6_K:  450 tensors
print_info: file format = GGUF V3 (latest)
print_info: file type   = Q6_K
print_info: file size   = 27,62 GiB (6,56 BPW) 
load: special_eos_id is not in special_eog_ids - the tokenizer config may be incorrect
load: printing all EOG tokens:
load:   - 2 ('<seed:eos>')
load: special tokens cache size = 128
load: token to piece cache size = 0,9296 MB
print_info: arch             = seed_oss
print_info: vocab_only       = 0
print_info: n_ctx_train      = 524288
print_info: n_embd           = 5120
print_info: n_layer          = 64
print_info: n_head           = 80
print_info: n_head_kv        = 8
print_info: n_rot            = 128
print_info: n_swa            = 0
print_info: is_swa_any       = 0
print_info: n_embd_head_k    = 128
print_info: n_embd_head_v    = 128
print_info: n_gqa            = 10
print_info: n_embd_k_gqa     = 1024
print_info: n_embd_v_gqa     = 1024
print_info: f_norm_eps       = 0,0e+00
print_info: f_norm_rms_eps   = 1,0e-06
print_info: f_clamp_kqv      = 0,0e+00
print_info: f_max_alibi_bias = 0,0e+00
print_info: f_logit_scale    = 0,0e+00
print_info: f_attn_scale     = 0,0e+00
print_info: n_ff             = 27648
print_info: n_expert         = 0
print_info: n_expert_used    = 0
print_info: causal attn      = 1
print_info: pooling type     = 0
print_info: rope type        = 2
print_info: rope scaling     = linear
print_info: freq_base_train  = 10000000,0
print_info: freq_scale_train = 1
print_info: n_ctx_orig_yarn  = 524288
print_info: rope_finetuned   = unknown
print_info: model type       = 36B
print_info: model params     = 36,15 B
print_info: general.name     = Seed-Oss-36B-Instruct
print_info: vocab type       = BPE
print_info: n_vocab          = 155136
print_info: n_merges         = 154737
print_info: BOS token        = 0 '<seed:bos>'
print_info: EOS token        = 2 '<seed:eos>'
print_info: PAD token        = 1 '<seed:pad>'
print_info: LF token         = 326 'Ċ'
print_info: EOG token        = 2 '<seed:eos>'
print_info: max token length = 1024
load_tensors: loading model tensors, this can take a while... (mmap = false)
load_tensors: offloading 64 repeating layers to GPU
load_tensors: offloading output layer to GPU
load_tensors: offloaded 65/65 layers to GPU
load_tensors:      Vulkan0 model buffer size = 27664,41 MiB
load_tensors:          CPU model buffer size =   621,39 MiB
.................................................................................................
llama_context: constructing llama_context
llama_context: n_seq_max     = 1
llama_context: n_ctx         = 4096
llama_context: n_ctx_per_seq = 4096
llama_context: n_batch       = 2048
llama_context: n_ubatch      = 512
llama_context: causal_attn   = 1
llama_context: flash_attn    = 1
llama_context: kv_unified    = false
llama_context: freq_base     = 10000000,0
llama_context: freq_scale    = 1
llama_context: n_ctx_per_seq (4096) < n_ctx_train (524288) -- the full capacity of the model will not be utilized
llama_context: Vulkan_Host  output buffer size =     0,59 MiB
llama_kv_cache:    Vulkan0 KV buffer size =  1024,00 MiB
llama_kv_cache: size = 1024,00 MiB (  4096 cells,  64 layers,  1/1 seqs), K (f16):  512,00 MiB, V (f16):  512,00 MiB
llama_context:    Vulkan0 compute buffer size =   313,00 MiB
llama_context: Vulkan_Host compute buffer size =   202,02 MiB
llama_context: graph nodes  = 2183
llama_context: graph splits = 258
common_init_from_params: added <seed:eos> logit bias = -inf
common_init_from_params: setting dry_penalty_last_n to ctx_size = 4096
main: llama threadpool init, n_threads = 16
main: chat template is available, enabling conversation mode (disable it with -no-cnv)
main: chat template example:
<seed:bos>system
You are a helpful assistant<seed:eos><seed:bos>user
Hello<seed:eos><seed:bos>assistant
Hi there<seed:eos><seed:bos>user
How are you?<seed:eos><seed:bos>assistant


system_info: n_threads = 16 (n_threads_batch = 16) / 32 | CPU : SSE3 = 1 | SSSE3 = 1 | AVX = 1 | AVX2 = 1 | F16C = 1 | FMA = 1 | BMI2 = 1 | AVX512 = 1 | AVX512_VBMI = 1 | AVX512_VNNI = 1 | LLAMAFILE = 1 | OPENMP = 1 | REPACK = 1 | 

main: interactive mode on.
sampler seed: 2243664103
sampler params: 
        repeat_last_n = 64, repeat_penalty = 1,200, frequency_penalty = 0,000, presence_penalty = 0,000
        dry_multiplier = 0,000, dry_base = 1,750, dry_allowed_length = 2, dry_penalty_last_n = 4096
        top_k = 20, top_p = 0,950, min_p = 0,000, xtc_probability = 0,000, xtc_threshold = 0,100, typical_p = 1,000, top_n_sigma = -1,000, temp = 0,600
        mirostat = 0, mirostat_lr = 0,100, mirostat_ent = 5,000
sampler chain: logits -> logit-bias -> penalties -> dry -> top-n-sigma -> top-k -> typical -> top-p -> min-p -> xtc -> temp-ext -> dist 
generate: n_ctx = 4096, n_batch = 2048, n_predict = -1, n_keep = 0

== Running in interactive mode. ==
 - Press Ctrl+C to interject at any time.
 - Press Return to return control to the AI.
 - To return control without starting a new line, end your input with '/'.
 - If you want to submit another line, end your input with '\'.
 - Not using system message. To change it, set a different value via -sys PROMPT


> hello
/home/runner/work/llama.cpp/llama.cpp/ggml/src/ggml-cpu/ggml-cpu.c:1942: GGML_ASSERT(t == 0 || t == 1) failed
/home/runner/work/llama.cpp/llama.cpp/ggml/src/ggml-cpu/ggml-cpu.c:1942: GGML_ASSERT(t == 0 || t == 1) failed
/home/runner/work/llama.cpp/llama.cpp/ggml/src/ggml-cpu/ggml-cpu.c:1942: GGML_ASSERT(t == 0 || t == 1) failed
/home/runner/work/llama.cpp/llama.cpp/ggml/src/ggml-cpu/ggml-cpu.c:1942: GGML_ASSERT(t == 0 || t == 1) failed
/home/runner/work/llama.cpp/llama.cpp/ggml/src/ggml-cpu/ggml-cpu.c:1942: GGML_ASSERT(t == 0 || t == 1) failed
/home/runner/work/llama.cpp/llama.cpp/ggml/src/ggml-cpu/ggml-cpu.c:1942: GGML_ASSERT(t == 0 || t == 1) failed
/home/runner/work/llama.cpp/llama.cpp/ggml/src/ggml-cpu/ggml-cpu.c:1942: GGML_ASSERT(t == 0 || t == 1) failed
/home/runner/work/llama.cpp/llama.cpp/ggml/src/ggml-cpu/ggml-cpu.c:1942: GGML_ASSERT(t == 0 || t == 1) failed
/home/runner/work/llama.cpp/llama.cpp/ggml/src/ggml-cpu/ggml-cpu.c:1942: GGML_ASSERT(t == 0 || t == 1) failed
/home/runner/work/llama.cpp/llama.cpp/ggml/src/ggml-cpu/ggml-cpu.c:1942: GGML_ASSERT(t == 0 || t == 1) failed
/home/runner/work/llama.cpp/llama.cpp/ggml/src/ggml-cpu/ggml-cpu.c:1942: GGML_ASSERT(t == 0 || t == 1) failed
/home/runner/work/llama.cpp/llama.cpp/ggml/src/ggml-cpu/ggml-cpu.c:1942: GGML_ASSERT(t == 0 || t == 1) failed
/home/runner/work/llama.cpp/llama.cpp/ggml/src/ggml-cpu/ggml-cpu.c:1942: GGML_ASSERT(t == 0 || t == 1) failed
/home/runner/work/llama.cpp/llama.cpp/ggml/src/ggml-cpu/ggml-cpu.c:1942: GGML_ASSERT(t == 0 || t == 1) failed
/home/runner/work/llama.cpp/llama.cpp/ggml/src/ggml-cpu/ggml-cpu.c:1942: GGML_ASSERT(t == 0 || t == 1) failed
/home/runner/work/llama.cpp/llama.cpp/ggml/src/ggml-cpu/ggml-cpu.c:1942: GGML_ASSERT(t == 0 || t == 1) failed
Could not attach to process.  If your uid matches the uid of the target
process, check the setting of /proc/sys/kernel/yama/ptrace_scope, or try
again as the root user.  For more details, see /etc/sysctl.d/10-ptrace.conf
ptrace: Operation not permitted.
No stack.
The program is not being run.
Could not attach to process.  If your uid matches the uid of the target
process, check the setting of /proc/sys/kernel/yama/ptrace_scope, or try
again as the root user.  For more details, see /etc/sysctl.d/10-ptrace.conf
ptrace: Operation not permitted.
No stack.
The program is not being run.
Could not attach to process.  If your uid matches the uid of the target
process, check the setting of /proc/sys/kernel/yama/ptrace_scope, or try
again as the root user.  For more details, see /etc/sysctl.d/10-ptrace.conf
ptrace: Operation not permitted.
No stack.
The program is not being run.
Could not attach to process.  If your uid matches the uid of the target
process, check the setting of /proc/sys/kernel/yama/ptrace_scope, or try
again as the root user.  For more details, see /etc/sysctl.d/10-ptrace.conf
ptrace: Operation not permitted.
No stack.
The program is not being run.
Could not attach to process.  If your uid matches the uid of the target
process, check the setting of /proc/sys/kernel/yama/ptrace_scope, or try
again as the root user.  For more details, see /etc/sysctl.d/10-ptrace.conf
ptrace: Operation not permitted.
No stack.
The program is not being run.
Could not attach to process.  If your uid matches the uid of the target
process, check the setting of /proc/sys/kernel/yama/ptrace_scope, or try
again as the root user.  For more details, see /etc/sysctl.d/10-ptrace.conf
ptrace: Operation not permitted.
No stack.
The program is not being run.
Could not attach to process.  If your uid matches the uid of the target
process, check the setting of /proc/sys/kernel/yama/ptrace_scope, or try
again as the root user.  For more details, see /etc/sysctl.d/10-ptrace.conf
ptrace: Operation not permitted.
No stack.
The program is not being run.
Could not attach to process.  If your uid matches the uid of the target
process, check the setting of /proc/sys/kernel/yama/ptrace_scope, or try
again as the root user.  For more details, see /etc/sysctl.d/10-ptrace.conf
ptrace: Operation not permitted.
No stack.
The program is not being run.
Could not attach to process.  If your uid matches the uid of the target
process, check the setting of /proc/sys/kernel/yama/ptrace_scope, or try
again as the root user.  For more details, see /etc/sysctl.d/10-ptrace.conf
ptrace: Operation not permitted.
No stack.
The program is not being run.
Could not attach to process.  If your uid matches the uid of the target
process, check the setting of /proc/sys/kernel/yama/ptrace_scope, or try
again as the root user.  For more details, see /etc/sysctl.d/10-ptrace.conf
ptrace: Operation not permitted.
No stack.
The program is not being run.
Could not attach to process.  If your uid matches the uid of the target
process, check the setting of /proc/sys/kernel/yama/ptrace_scope, or try
again as the root user.  For more details, see /etc/sysctl.d/10-ptrace.conf
ptrace: Operation not permitted.
No stack.
The program is not being run.
Aborted (core dumped)
(base) andrey@halo:~/Downloads/build/bin$ Unable to attach: program terminated with signal SIGABRT, Aborted.
```

**Comment by CISC** [comment]
That's odd, you seem to have triggered a `FLASH_ATTN_BACK` op, which shouldn't be possible. Just in case though, try without `-fa`.

**Comment by ndrewpj** [comment]
bug not seen in latest releases, closing.
